frontend/src/components/common/Resource/Resource.tsx

```diff
@@ -667,7 +667,7 @@ export function OwnedPodsSection(props: OwnedPodsSectionProps) {
 
   const queryData = {
     namespace: resource.kind === 'Namespace' ? resource.metadata.name : undefined,
-    labelSelector: labelSelectorToQuery(resource?.spec?.selector),
+    labelSelector: resource?.spec?.selector ? labelSelectorToQuery(resource?.spec?.selector) : '',
```

Comment:
Ah. The type checker is turned off for spec(it has an any type). All those any like `[otherProps: string]: any;` in cluster.ts are doing us a disservice.